.github/workflows/build-coreeval.yml

```diff
@@ -0,0 +1,94 @@
+name: Build CoreEval
```

Comment:
this isn't building a CoreEval, just bundles.

This is worth landing but please change the name to "Build Package Bundles" and update the workflow filename.

Reply:
The workflow file name still needs updating to build-bundles